api/utils/index.js

```diff
@@ -205,18 +205,23 @@ function omitBy(fn, obj) {
   return pick(pickedKeys, obj);
 }
 
-async function paginate(model, serialize, params, query = {}) {
+async function paginate(
+  model,
+  serialize,
+  params,
+  query = {},
+  order = ['createdAt', 'DESC'],
+) {
   const limit = toInt(params.limit) || 25;
   const page = toInt(params.page) || 1;
   const offset = limit * (page - 1);
 
   const pQuery = {
     limit,
     offset,
-    order: [['createdAt', 'DESC']],
+    order: [order],
```

Comment:
This might have already been incorrect but `api/admin/controllers/domain.js` uses the paginate function twice with `order` supplied in the `query`; I'd check that this doesn't create two separate sorts

Reply:
I'll take a look

Reply:
It is just overwriting the preset `order` if the `order` is added in the query params. I'm going to move the order into the `query` params.